includes/exceptions.h

```diff
@@ -29,21 +35,23 @@ struct Exception : std::exception {
   OrtErrorCode code_;
 };
 
-#if defined(OCOS_NO_EXCEPTIONS) || defined(OCOS_PREVENT_EXCEPTION_PROPAGATION)
-inline void PrintFinalMessage(const char* file, int line, const char* msg) {
+// helper that outputs an error message in a platform aware manner
+// Usages:
+//  - logging exception message when they may not propagate up
+//  - logging failure when using the ORT logger
+inline void LogError(const ORTCHAR_T* file, int line, const char* msg) {
 #if defined(__ANDROID__)
-  __android_log_print(ANDROID_LOG_ERROR, "onnxruntime-extensions", "Exception in %s line %d: %s", file, line, msg);
+  __android_log_print(ANDROID_LOG_ERROR, "onnxruntime-extensions", "Error in %s line %d: %s", file, line, msg);
 #else
-  std::cerr << "Exception in " << file << " line " << line << ": " << msg << std::endl;
+  std::cerr << "Error in " << file << " line " << line << ": " << msg << std::endl;
```

Comment:
may need to convert from `const wchar_t*` to `const char*` first if ORTCHAR_T is wchar_t

Reply:
huh, didn't know that `std::wostream` can work with `char*` too